src/core/slices/webuiSettings.ts

```diff
@@ -78,34 +80,34 @@ export const defaultLayout = {
     }, {
       i: 'tvdb', x: 0, y: 9, w: 4, h: 5, minW: 3, minH: 5, maxH: 10, moved: false, static: false,
     }],
-  },
+  } as LayoutType,
   settings: {
     lg: [{
-      i: 'general', x: 0, y: 0, w: 4, h: 15, minW: 3, minH: 5, moved: false, static: false,
+      i: 'general', x: 0, y: 0, w: 4, h: 22, minW: 3, minH: 5, moved: false, static: false,
     }, {
-      i: 'anidb-login', x: 0, y: 15, w: 4, h: 8, minW: 3, minH: 5, moved: false, static: false,
+      i: 'anidb-login', x: 0, y: 22, w: 4, h: 11, minW: 3, minH: 5, moved: false, static: false,
     }, {
-      i: 'relation', x: 0, y: 23, w: 4, h: 10, minW: 3, minH: 3, moved: false, static: false,
+      i: 'relation', x: 0, y: 33, w: 4, h: 15, minW: 3, minH: 3, moved: false, static: false,
     }, {
-      i: 'anidb', x: 4, y: 0, w: 4, h: 21, minW: 3, minH: 5, moved: false, static: false,
+      i: 'anidb', x: 4, y: 0, w: 4, h: 28, minW: 3, minH: 5, moved: false, static: false,
     }, {
-      i: 'import', x: 4, y: 21, w: 4, h: 7, minW: 3, minH: 3, moved: false, static: false,
+      i: 'import', x: 4, y: 28, w: 4, h: 11, minW: 3, minH: 3, moved: false, static: false,
     }, {
-      i: 'plex', x: 4, y: 28, w: 4, h: 5, minW: 3, minH: 3, moved: false, static: false,
+      i: 'plex', x: 4, y: 39, w: 4, h: 9, minW: 3, minH: 3, moved: false, static: false,
     }, {
-      i: 'tvdb', x: 8, y: 0, w: 4, h: 12, minW: 3, minH: 5, moved: false, static: false,
+      i: 'tvdb', x: 8, y: 0, w: 4, h: 17, minW: 3, minH: 5, moved: false, static: false,
     }, {
-      i: 'moviedb', x: 8, y: 12, w: 4, h: 7, minW: 3, minH: 5, moved: false, static: false,
+      i: 'moviedb', x: 8, y: 17, w: 4, h: 10, minW: 3, minH: 5, moved: false, static: false,
     }, {
-      i: 'language', x: 8, y: 19, w: 4, h: 9, minW: 3, minH: 3, moved: false, static: false,
+      i: 'language', x: 8, y: 27, w: 4, h: 12, minW: 3, minH: 3, moved: false, static: false,
     }, {
-      i: 'trakt', x: 8, y: 28, w: 4, h: 5, minW: 3, minH: 3, moved: false, static: false,
+      i: 'trakt', x: 8, y: 39, w: 4, h: 9, minW: 3, minH: 3, moved: false, static: false,
     }],
-  },
+  } as LayoutType,
 };
 
 const initialState = {
-  v3: {
+  webui_v2: {
```

Comment:
same with regard to server managing settings versioning, should not be necessary, also maybe just use key name like "server" to indicate it is managed by server. I am thinking of possible clash during migration and what not maybe version number is inside the server response then we can at least put some code like - you are using webui v1, you should use v2 etc. in case server gives different settings object.